clients/onboarding/src/pages/company/OnboardingCompanyRegistration.tsx

```diff
@@ -325,7 +325,33 @@ export const OnboardingCompanyRegistration = ({
                             onPress={() => onChange(!value)}
                             style={styles.tcuCheckbox}
                           >
-                            <LakeCheckbox value={value} isError={isNotNullish(error)} />
+                            <Box direction="row" alignItems="center">
```

Comment:
nit: prefer adding `flexDirection: row and alignItems: center` to `styles.tcuCheckbox` instead of adding an extra `div`